index.js

```diff
@@ -1,39 +1,76 @@
-const bookList = document.querySelector('#bookList');
-const addBtn = document.getElementById('addBtn');
-let collection = JSON.parse(localStorage.getItem('bookCollection')) || [];
-
-function renderCollection() {
-  let html = '';
-  collection.forEach((element, index) => {
-    const removeBtn = `<button onclick='removeBook(${index})'>Remove</button>`;
-    html += `<div class='book'>
-        <div>${element.title}</div>
-        <div>${element.author}</div>
-        ${removeBtn}
-        <hr>
-        </div>`;
-  });
-  bookList.innerHTML = html;
-}
+/* eslint-disable no-use-before-define */
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ]  Kindly don't disable the linters since they are very important to improve your code and make it DRY KISS and clean since it is about some rules to organize your code so kindly don't disable them and try to solve the issues instead of disabling them✨🚀💯😊🔥